packages/salesforcedx-utils-vscode/package.json

```diff
@@ -16,7 +16,6 @@
     "applicationinsights": "1.0.7",
     "cross-spawn": "7.0.3",
     "rxjs": "^5.4.1",
-    "strip-ansi": "^5.2.0",
```

Comment:
I removed strip-ansi because:
* I needed it in salesforcedx-utils, but including it as a dep caused build issues at strip-ansi@5
* I tried bumping to 6+, but strip-ansi is now ESM

As an alternative I grabbed the regex used by strip-ansi, along with proper attribution (MIT), and included it in our source, with necessary cover functions to reach the desired functionality and use cases.